### ISSUE TITLE
Toast close button does not have a sufficient focus style

### ISSUE THREAD

**Issue by jnurthen** [adobe/spectrum-web-components#753]
### Expected Behaviour
The focus style should be discernable to people with low vision

### Actual Behaviour
The focus style is too subtle
![Screen Shot showing the keyboard focus style on the toast close button](https://user-images.githubusercontent.com/1724479/87099012-888c3c80-c1fd-11ea-9167-f37075ba68b7.png)

#### Steps to Reproduce
* https://opensource.adobe.com/spectrum-web-components/components/toast
* tab through the page until you reach the close button

#### Platform and Version
Chrome 83
Safari 14
FF 78

**Comment by Westbrook** [comment]
Luckily, it seems that this will come for free with the upgrade to Spectrum CSS 3.0.0 for the `button` package! Check it out in development here: https://5f0b5f499b119400073dfbf6--spectrum-web-components.netlify.app/components/toast Yay. 🎆
